ivy/functional/ivy/general.py

```diff
@@ -1826,7 +1826,7 @@ def current_backend_str() -> Union[str, None]:
     """
     fw = current_backend()
     if not backend_stack:
-        return ""
+        return ivy.utils.backend.handler.implicit_backend
```

Comment:
ig we shouldn't make this change for the reason mentioned in the `__init__.py` comment